addon-test-support/@ember/test-helpers/setup-context.ts

```diff
@@ -139,8 +144,9 @@ export default function(context, options: any = {}) {
     .then(() => {
       let application = getApplication();
       if (application) {
-        return application.boot();
+        return application.boot().then(() => {});
```

Comment:
This seems somewhat odd, are we just trying to make sure it doesn’t return something?

Reply:
yes, the API contract is that we return `void`. we could change it to `Promise<any>` but I'm not sure if that's a good idea 🤔

Reply:
Yep, totally agree with you.